.cirrus.yml

```diff
@@ -23,6 +23,9 @@ env:
     SCRIPT_BASE: "./contrib/cirrus"
     PACKER_BASE: "./contrib/cirrus/packer"
     CIRRUS_CLONE_DEPTH: 200
+    # Command to prefix every output line with a timestamp
+    # (can't do inline awk script, Cirrus-CI or YAML mangles quoting)
+    TS: "awk --file ${CIRRUS_WORKING_DIR}/${SCRIPT_BASE}/ts.awk"
```

Comment:
Would you consider spelling it out, at least in the filename? `timestamp.awk` is more readable and findable.

Reply:
sure sure, no prob.